bpfd-operator/config/default/patch.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+  annotations:
+    openshift.io/node-selector: ""
+    openshift.io/description: "Openshift bpfd components"
+    workload.openshift.io/allowed: "management"
+  name: system
```

Comment:
This actually shouldn't be here, but in general this patch just adds OCP specific annotation to the bpfd namespace.. It's confusing because the main kustomize.yaml replaces all namespace names from `system` to `bpfd`

Reply:
Done